source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,169 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited : 
+
+
+Avaliable timeouts
+------------------
+
+Captive portal has 3 integrated timeout options :
+
+:**Idle timeout**: Users will automatically disconnected after a defined amount of network inactivity.
+  The "last activity time of each user" is retrived using the last byte sent by each device. 
+  (meaning last byte "uploaded" from the user perspective)
+:**Hard timeout**: Users will be automatically disconnected after a defined time period, whether or not they are active.
+:**Traffic quota**: Users will be automatically disconnected after exceeding a defined amount of combined upload and download transfer data.
+
+
+If one of these options is enabled, pfSense will check once per minute to find users that meet criteria for disconnection.
+The way pfSense works is that a scheduled task (cron) will run every minute, checking for users to disconnect.
+
+.. tip:: These options disconnect users but they do not prevent users from immediately logging in again.
+ To prevent users from logging in again, use a RADIUS server for authentication with appropriately configured limits. See :ref:`label-radius-server`
+
+Authentication methods
+----------------------
+
+Four authentication methods are avaliable on pfSense :
+
+:Don't authenticate users: All users will be redirected to the captive portal
+  login page but will get connected as soon as they click **Login**. this is useful for
+  displaying a welcome page, or forcing users to see general terms and conditions.
+:Use an authentication server from the user manager: Authentication will be made using a remote authentication server. Please see :ref:`label-user-manager` for details.
+:Use RADIUS MAC Authentication: This authentication method emulate 802.1X, allowing some MAC addresses to get automatically connected.
```

Comment:
Should be `emulates`.

Also consider changing `some MAC addresses to get automatically connected` to `some MAC address to connect automatically`.